tests/v2/validation/charts/gatekeeperconfig.go

```diff
@@ -0,0 +1,55 @@
+package charts
+
+import "gopkg.in/yaml.v2"
+
+type ConfigYaml struct {
```

Comment:
@nataliefyi do you think `gatekeeperconfig.go` and `opaconstraint.go` should be helpers rather than strictly be here? Would like to hear your opinion. Could these be extended to future tests? Or are they strictly for gatekeeper allowed namespaces?

Reply:
moved these to extensions